docs/tutorials/getting-started-eclipse.md

```diff
@@ -0,0 +1,71 @@
+---
+type: tutorial
+layout: tutorial
+title:  "Getting Started with Eclipse Luna"
+description: "This tutorial guides through installing Kotlin plugin for Eclipse and creating Hello World application."
+authors: Nikolay Krasko
+date: 2015-03-28
+showAuthorInfo: true
+---
+### Setting up the environment
+We will show how to install Kotlin plugin in Eclipse Luna SR2 (4.4.2) and create a simple Hello World application.
+
+1. Start from getting and installing [Eclipse Luna](https://www.eclipse.org/downloads/). "Eclipse IDE for Java Developers" 
```

Comment:
Steps for other tutorials usually don't have numbers. Would be nice to use same style. Also the step name would be better with "Start from". It's already a step.